TeknoParrotUi.Common/GameProfiles/GGXX.xml

```diff
@@ -2,13 +2,13 @@
 <GameProfile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <GameName>Guilty Gear XX Accent Core Plus R</GameName>
   <GamePath/>
-  <TestMenuParameter>-t</TestMenuParameter>
+  <TestMenuParameter></TestMenuParameter>
```

Comment:
same as for other game, how do you access test menu then? (and if its ingame then change hasseparatetestmode)

Reply:
OK, re-issued.